Cargo.toml

```diff
@@ -74,7 +68,6 @@ serialize-serde = ["serde"]
 # purposes. New code should use none of these features.
 coresymbolication = []
 dbghelp = []
-dl_iterate_phdr = []
```

Comment:
This feature itself should probably remain for back compat reasons. It would just do nothing.

Reply:
Thank you for your review, I think it makes sense.

Reply:
Thanks for your review. 

`dl_iterate_phdr` is a dev dependency, and users will not explicitly specify it. I think it can be removed.